Dockerfile

```
@@ -1,4 +1,4 @@
-FROM amazoncorretto:8-alpine3.14-jre
+FROM amazoncorretto:11.0.16-alpine3.15
```

We want the jre only without all the java compiler and other utilities that are in the JDK image.
Unfortunately there is no Java 11 amazoncorretto jre image so I would just bump the alpine version:
```suggestion
FROM amazoncorretto:8-alpine3.15-jre
```

---

Since you need Java 11 for checkstyle we can try to migrate to eclipse-temurin distribution:
```suggestion
FROM eclipse-temurin:11-jre-alpine
```